### PR TITLE
CI-14523 added support for hardware acceleration in CI

### PR DESCRIPTION
**Pull request by nofarblue**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: added support for hardware acceleration on harness CI cloud linux/arm machines
* Jira/GitHub Issue numbers (if any): CI-14523
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/use-harness-cloud-build-infrastructure.md [hunk 1]
```diff
@@ -165,10 +165,40 @@ To select a resource class size, please modify the CI stge cloud infrastructure
...
+
+Harness supports hardware acceleration using nested virtualization on Linux/AMD Cloud machines.
+
+By enabling this feature, Android SDK tools and emulators can run more efficiently within virtualized environments, making Android test execution faster and optimizing build time.
+
+To enable this feature, set the `nestedVirtualization` property to `true` as shown below. 
+
+:::note
+* To enable `nestedVirtualization` use the YAML editor, as this option is currently unavailable in the visual editor.
+* Hardware acceleration is available for all machine sizes, when using Linux with ARM architecture on Harness Cloud.
```
Comment (ShobhitSingh11) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/use-harness-cloud-build-infrastructure.md:182]: Linux with AMD architecture

**Review by ShobhitSingh11 — DISMISSED** [review]

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/use-harness-cloud-build-infrastructure.md (hunk 1 above)
Comment (nofarblue) [reply docs/continuous-integration/use-ci/set-up-build-infrastructure/use-harness-cloud-build-infrastructure.md]: fixed thanks

**Review by dewan-ahmed — DISMISSED** [review]
Requested changes done via https://github.com/harness/developer-hub/pull/8439/commits/ac904cd5f92596113797083057b7e81770bba90d

LGTM

**Review by dewan-ahmed — APPROVED** [review]
LGTM

**Pull request merged**